src/components/views/auth/PasswordLogin.tsx

```diff
@@ -433,7 +433,8 @@ export default class PasswordLogin extends React.PureComponent<IProps, IState> {
         const loginField = this.renderLoginField(this.state.loginType, !autoFocusPassword);
 
         let loginType;
-        if (!SdkConfig.get().disable_3pid_login) {
+        const disable_3pid_login =SdkConfig.get().loginScreen?.disable_3pid_login??false;
```

Comment:
Please revert this change and continue to use the element default config location for disable_3pid_login

Reply:
I am not sure why to revert this change. there are multiple configurations inside the login screen which i have kept into the login screen config. it will be easier for a person to identify the configuration by that.
"loginScreen": {
"displaySecondaryLogo":false,
"displaySecondaryVerticalLogo":false,
"displayLanguageSelector":false,
"disable_3pid_login": false,
"showHomeServerInfo": false,
"showMatrixDefaultFooterLinks": false,
"allowPhoneBasedAuthentication": false,
"allowEmailBasedAuthentication": false,
"changeSigninToLoginTextLabel": true,
"backgroundColor": "(0,38, 119)",
"logo":{
"img":{
"width": "150px",
"height": "100px"
}
}

Reply:
Because you are changing the default element behaviour, which we are trying to avoid

Reply:
done